social-distancing/ui/WebGUI.py

```diff
@@ -5,28 +5,52 @@
 from flask import render_template
 from flask import Response
 
-from .utils import visualization_util as vis_util
+from .utils import visualization_utils as vis_util
+
+category_index = {
+    "id": 0,
+    "name": "safe",
+    "id": 1,
+    "name": "unsafe",
+}  # TODO: json file for detector config
 
-class WebGUI:
 
+class WebGUI:
     def __init__(self, config, engine_instance):
         self.config = config
         self.__ENGINE_INSTANCE = engine_instance
         self._output_frame = None
         self._lock = threading.Lock()
         self.app = self.create_flask_app()
 
-    def update(self, input_frame, boxes, distances):
-        # TODO: docstring after completing
-        # TODO: using vis_util for visualizing
+    def update(self, input_frame, nn_out, distances):
```

Comment:
I did not see you changed this, DO NOT touch the API!

Reply:
Oh actually this is a great naming as it stands for neural network's output. Okay :D